content/blog/keda-2.0-beta.md

```diff
@@ -58,7 +58,7 @@ With our official release we will provide [migration scripts](https://github.com
 > 
 > KEDA comes with a metrics server and Kubernetes only allows you to run one of them in a cluster.
 > 
->_Learn more about how KEDA is architected in [our docs](http://keda.sh/docs/latest/concepts/#architecture)._
+>__Learn more about how KEDA is architected in [our docs](http://keda.sh/docs/latest/concepts/#architecture).__
```

Comment:
The goal is to have italic section which is not fixed yet. Can you try locally please or check the PR preview?

Reply:
<img width="919" alt="Screenshot 2020-09-11 at 20 41 25" src="https://user-images.githubusercontent.com/9528307/92961223-3242b080-f46f-11ea-9035-aefd79d0e877.png">